src/angular/11-declarative-state/declarative-state.md

```diff
@@ -148,33 +141,44 @@ To go more in depth about this topic check out these articles:
 
 ## Exercise: Update states, cities & restaurants to be observables
 
-### The Problem
+### The Problem 
+
+Convert the imperatively managed state in the restaurant component to declarative state. 
 
-Convert the imperatively managed state in the restaurant component to declarative state.
+When you're finished the component members `state`, `cities` & `restaurants` will be of the types `Observable<Data<State>>`, 
+`Observable<Data<City>>` and `Observable<Data<Restaurant>>` respectively. Each will be defined as a set of RxJS 
+operators that either produce values from a response emitted by a service layer request, or produce values from changes 
+in a form control (which in turn may make a request).
 
-When you're finished the component members `state`, `cities` & `restaurants` will be of the types `Observable<Data<State>>`,
-`Observable<Data<City>>` and `Observable<Data<Restaurant>>` respectively. Each will be defined as a set of RxJS
-operators that produce the type from a response emitted by a service layer request or an event emitted by a form
-control (which in turn may make a request).
+Since we're no longer using Reactive Forms, to get value streams from a form control you'll use `BehaviorSubjects` 
+which have `.next($event.target.value)` called on them during control `change` events.
+
+You'll also add new streams to handle several functions that were previously handled by the imperative Reactive 
+Forms API:
+- `displayedCity` which sets the city select control value to user input if there's been any, or clears it if the state select control has been changed
+- `stateSelectDisabled` which disables the state select control if loading is ongoing
+- `citySelectDisabled` which disables the city select control if loading is ongoing or no values are available
 
 ### What You Need To Know
 
 - How to perform common RxJS operations like:
     - setting the initial value to be emitted
     - transforming a value emitted
-    - emit values into a stream from another stream
-    - take action based on a value as it emitted by a stream
-    - multicasting emissions of a "cold" observable and handling late subscribers
-
- You've learnt all of the above as part of the earlier sections on this page! Completing the
- [Bitovi Academy's RxJS training](/learn-rxjs.html) may help however.
-
+    - conditionally emit values into a stream from another stream
+    - merge streams into a single stream of values 
+    - merge streams into a stream of tuples
```

Comment:
unless the reader has specific TypeScript knowledge, they may not know the term "tuple". I don't think we define it anywhere in the Angular training previously, not sure if we should worry about that?

Reply:
i could rewrite that as "merge streams into a stream of arrays with values from each input stream"